xmtp_mls/Cargo.toml

```diff
@@ -48,6 +48,7 @@ toml = "0.7.4"
 tracing = "0.1.37"
 xmtp_cryptography = { path = "../xmtp_cryptography" }
 xmtp_proto = { path = "../xmtp_proto", features = ["proto_full"] }
+xmtp_v2 = { path = "../xmtp_v2" }
```

Comment:
I think once we get to shaving fat off the WASM build we're going to want to extract the functions we need from this crate to avoid bloat. Or get really aggressive with features.

Problem for another day